lib/uktt/http.rb

```diff
@@ -17,7 +17,8 @@ def initialize(connection, options)
 
     def retrieve(resource, query_config = {})
       resource = File.join(service, 'api', version, resource) if public?
-      resource = "/#{resource}#{query_params(query_config)}"
+      resource = File.join('/', resource)
+      resource = "#{resource}#{query_params(query_config)}"
```

Comment:
Interesting!

Reply:
interesting indeed.  It felt odd and we discussed and troubleshoot it a bit. 
But it didn't look like it was the root cause in the end.

Reply:
Yeah it seems like it drops all the params when you have `//`